tst/providers/MeetingProvider/index.test.tsx

```diff
@@ -18,17 +19,21 @@ import { MeetingManagerJoinOptions } from '../../../src/providers/MeetingProvide
 import '@testing-library/jest-dom';
 
 import { act, renderHook } from '@testing-library/react';
-import React from 'react';
 
-import { MeetingProvider, useMeetingManager } from '../../../src/providers/MeetingProvider';
+import {
+  MeetingProvider,
+  useMeetingManager,
+} from '../../../src/providers/MeetingProvider';
 
 describe('Meeting Provider', () => {
   it('events are received correctly', async () => {
     // Mock the user agent to ensure the MeetingManager event subscription is
     // set up. Otherwise, the amazon-chime-sdk-js will not detect a valid
     // browser to use.
     const userAgentGet = jest.spyOn(navigator, 'userAgent', 'get');
-    userAgentGet.mockReturnValue('Chrome/96.0');
+    userAgentGet.mockReturnValue(
```

Comment:
Only change mock user agent to fix a unit test error. Rest change in this file are linting.